build/broccoli/build-packages.js

```diff
@@ -98,7 +98,7 @@ function treeForPackage(packagePath) {
 
   let packageJSONTree = treeForPackageJSON(packagePath);
 
-  let tree = merge([packageTree, packageJSONTree]);
+  let tree = merge([packageTree, packageJSONTree, funnel(process.cwd(), { files: ['LICENSE'], destDir: '/' })]);
```

Comment:
We could make a temp dir and copy it in, use broccoli-file-creator, maybe use an unwatcheddir, etc? Instead